db/schema.rb

```diff
@@ -31,11 +48,18 @@
     t.datetime "deleted_at"
   end
 
-  add_index "deploys", ["deleted_at"], name: "index_deploys_on_deleted_at", using: :btree
-  add_index "deploys", ["job_id", "deleted_at"], name: "index_deploys_on_job_id_and_deleted_at", using: :btree
-  add_index "deploys", ["stage_id", "deleted_at"], name: "index_deploys_on_stage_id_and_deleted_at", using: :btree
```

Comment:
If we want the btree index back, I'll have to add another migration to recreate the indexes directive specifying the btree option:
add_index :deploys, :stage_id, using: 'btree')